### PR TITLE
updated events

### PR DESCRIPTION
**Pull request by SooYou**
I took the time to update the socket event descriptions.

### CODE REVIEW

File: api/events/backend_events/userUpdate.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    'p': {
+        'i': 3865819,
+        'level': 12,
+        'avatarID': 'base01',
+        'username': 'SooYou',
+        'badge': '80sb01'
+    }, 
+    's': 'dashboard'
+}
+```
```
Comment (goto-bus-stop) [inline api/events/backend_events/userUpdate.md:63]: Maybe several "real" examples for a few of the different actual changes that you might receive? That will make it clearer that you have to account for the differences in object structure manually.

``` json
{
    "a": "userUpdate",
    "p": {
        "i": 245325,
        "avatarID": "base13"
    },
    "s": "tastycat"
}
```

``` json
{
    "a": "userUpdate",
    "p": {
        "i": 245325,
        "username": "I Changed My Username!"
    },
    "s": "tastycat"
}
```

Comment (SooYou) [inline api/events/backend_events/userUpdate.md:63]: agreed

**Comment by goto-bus-stop** [comment]
also, perhaps the examples could use ```json fenced code blocks instead, although then you do have to switch to double quotes as well for validity

OTOH, other examples in the repo have just used ```js as well so it's probably fine :smile:

**Comment by samhmills** [comment]
JS for frontend, JSON for backend I think is best, although it doesn't matter too much :smile: 

I say this because that is what the data types actually are

**Comment by goto-bus-stop** [comment]
Looks good now :shipit:

**Comment by SooYou** [comment]
yea I just went with the style that was already used for modBan for consistency :smile:

**Pull request merged**